src/lib/expression/pqp_select_expression.cpp

```diff
@@ -0,0 +1,46 @@
+#include "pqp_select_expression.hpp"
```

Comment:
Did you already think about how an optimizer rule would look like that transforms these subselects into joins?